packages/apps/composer-app/src/main.tsx

```diff
@@ -53,19 +54,14 @@ import { setupConfig } from './config';
 import { appKey } from './globals';
 import { steps } from './help';
 import { INITIAL_CONTENT, INITIAL_TITLE } from './initialContent';
-import { initializeNativeApp } from './native';
 import translations from './translations';
 
 const main = async () => {
   const config = await setupConfig();
   const services = await createClientServices(config);
-
   const isSocket = !!(globalThis as any).__args;
-  if (isSocket) {
-    void initializeNativeApp();
-  }
-
   const App = createApp({
+    
```

Comment:
```suggestion

  const App = createApp({
```

small nit about spacing